<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="IP">
            <property name="text">
             <string>IP:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ip_Servidor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="Connect">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Disconnect">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="Max">
            <property name="text">
             <string>Max</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_Max">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd_Max"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="Min">
            <property name="text">
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_Min">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd_Min"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="Timing">
            <property name="text">
             <string>Timing(s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_Timing">
            <property name="maximum">
             <number>3</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcd_Timing"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Start">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Stop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>horizontalLayoutWidget_3</zorder>
   <zorder>horizontalLayoutWidget_7</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>slider_Max</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_Max</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_Min</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_Min</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_Timing</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcd_Timing</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
